WalletWasabi/Blockchain/Transactions/SmartTransaction.cs

```diff
@@ -170,7 +170,6 @@ public IReadOnlyCollection<ForeignVirtualOutput> ForeignVirtualOutputs
 	[JsonProperty]
 	[JsonConverter(typeof(Uint256JsonConverter))]
 	public uint256? BlockHash { get; private set; }
```

Comment:
Added back separation line.

```suggestion
	public uint256? BlockHash { get; private set; }
	
```